### PR TITLE
wolfssl/4.4.0 add more set of configuration options

### PR DESCRIPTION
**Pull request by kirandivekar**
Added new set of configuration options which are
typically needed for packages like curl

Specify library name and version:  **lib/1.0**

- [ ] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [ ] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [ ] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [ ] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Review by danimtb — APPROVED** [review]

**Comment by kirandivekar** [comment]
Although, @prince-chrismc has approved the changes, need a review from member with write access. @uilianries can you please help with review?

**Pull request merged**
